MAINTAINERS

```diff
@@ -240,6 +256,11 @@ example and you'll be fine".
 		"stevvooe",
 	]
 
+	[Org.Projects]
```

Comment:
SGTM, figure people would do a pull request to the MAINTAINERS file here and propose which new projects to add, then the maintainers can actually vote on the PR.